agent-manager/config/agents/agent-1.yaml

```diff
@@ -0,0 +1,13 @@
+name: "agent-1"
+description: "Agent 1 is a helpful assistant that can answer questions and help with tasks."
+model: "llama3.2:3b"
+instructions: "You are a helpful agent"
+tool_choice: "auto"
+input_shields: []
+output_shields: []
+max_infer_iters: 10
+mcp_servers:
+  - "asset_database"
+  - "servicenow"
```

Comment:
We don't have these yet so we can't register them yet
```suggestion
```

Reply:
Applied the suggestion, thanks!